packages/fuel-gauge/src/reentrant-contract-calls.test.ts

```diff
@@ -54,7 +54,7 @@ describe('Reentrant Contract Calls', () => {
      * number 42 (from `Foo.foo`) instead of 1337 (from `Foo.baz`).
      */
     const returnReceipts = receipts.filter(
-      (r) => r.type === ReceiptType.Return && r.id === fooContract.id.toB256()
+      (r) => r.type === ReceiptType.ReturnData && r.id === fooContract.id.toB256()
```

Comment:
@danielbate Why did this change was necessary?

Reply:
Under `v1`, each call now creates a `ReturnData` receipt and the value comes from there rather than the `Return` reciept. Some logic needs to be removed from where we build the invocation result, but I will do that in #2109.